db/seeds/seeder.rb

```diff
@@ -0,0 +1,35 @@
+class Seeder
```

Comment:
It would be more idiomatic in Rails to add this to the autoload path, e.g. in `app/lib`.